File: Multiples of 3 and 5/multiples3And5.js
let number=prompt()

// Summation of multiples of 3 and 5
function multiplesOf3And5(number){
    let i=0,sum=0,sum1=0,final=0;
    while (i<number){
        if (i%3===0){
            // console.log(i)
            sum+=i;
        }
        if (i%5===0){
            sum1+=i;
            console.log(i)
        }
        ++i;
    }
    final=sum+sum1;
    console.log(`Summation of multiples of 3 and 5 in range ${number} is ${final}`)
    return final
}

multiplesOf3And5(number);

File: Multiples of 3 and 5/multiples3And5.py
number=int(input())

def multiplesOf3And5(number):
    i=0;
    sum0=0;
    sum1=0;
    sumBoth=0
    while i<number:
        # divisible by both
        if(i%3==0 and i%5==0):
            sumBoth+=i
        elif i%3==0:
            sum0+=i
            print(i)
            # print(sum0)
        elif i%5==0:
            sum1+=i
            print(i)
        i+=1
    final=sum0+sum1+sumBoth
    print("Summation of multiples of 3 and 5 is %d" %final)
    return final
            
multiplesOf3And5(number)